Multilingual.php

```diff
@@ -31,6 +31,57 @@ function redcap_data_entry_form($project_id, $record, $instrument){
 		echo '<script type="text/javascript">' . str_replace('APP_PATH_IMAGES', APP_PATH_IMAGES, str_replace('REDCAP_LANGUAGE_VARIABLE', $this->languageVariable($project_id), str_replace('REDCAP_AJAX_URL', $this->getUrl("index.php", true), file_get_contents($this->getModulePath() . 'js/multilingual.js')))) . '</script>';
 		echo '<link rel="stylesheet" type="text/css" href="' . $this->getUrl('css/multilingual.css') . '">';
 	}
+	
+	function redcap_pdf($project_id, $metadata, $data, $instrument, $record, $event_id, $instance) {
+		// delay execution of this module to allow multi-consent-signature module to do it's thing
+		if ($this->delayModuleExecution()) {
+			return;
+		}
+		
+		// decide which language this pdf needs to be translated to by looking for 'languages' field value in $data
+		$lang_field_name = $this->languageVariable($project_id);
+		if (empty($event_id)) {
+			// attempt to get event_id from data if not given as arg
+			$event_id = array_key_first($data[$record]);
+		}

```

Comment:
I'd only want to remove these lines if $event_id is guaranteed to exist in all scenarios.

Reply:
The docs do seem to suggest that `$event_id` can be null...  Off the top of my head I'm struggling to imagine a scenario where that would be possible though.  If there is such a scenario, I also wonder if getting the `$event_id` from the first data point is actually a safe assumption...  I think this would be a small edge case regardless, but it might be worth verifying if it's easy to do and the goal is to mitigate any risk introduced by these changes.  A quick question to Rob might clarify the scenario where `$event_id` is null.

Reply:
I just checked and, coincidentally, this section of code was re-written and setting $event_id in this way no longer occurs.